### DISCUSSION TITLE
slug=pathToPage instead of /lg/page

### DISCUSSION THREAD

**Discussion by helinW** [wintercms/wn-translate-plugin#53] [Q&A] [answered]
Hello, 
I have a multilingual website and I have a problem with the page url. I use winter.translate. When I'm in "english", all links are ok. For instance, I'm on the page : domain.com/contact. When I use the local picker to change the language, it's ok, I'm redirected on page domain.com/es/contact. But the url of the contact link become domain.com/es?slug=contact ansd all other links to domain.com/es?slug=page
Can you tell me what I did wrong or forget ? Thank you

**Comment by mjauvin** [comment]
Can you show the full content of you contact page? (specifically, the first part before the first "==")

**Comment by helinW** [comment]
title = "title.contact"
url = "/contact"
layout = "default"
is_hidden = 0

[viewBag]
localeUrl[fr] = "/contact"
localeUrl[es] = "/contact"

==

**Comment by mjauvin** [comment]
Did you customize the `localePicker` component markup or do you use the default markup?

The more details you give, the better we can help.

**Comment by helinW** [comment]
I customized the localpicker but it the same problem with the original component.
In fact when I change language, the website didn't find the page and redirect to the 404 page.
You can see there : [https://ossmozbjj.com/]
It's like the system didn't recognize the prefix of the page .

In my file .env, I add this :
TRANSLATE_PREFIX_LOCALE=false

**Comment by mjauvin** [comment]
How do you create the links on your pages (home/contact) ?

It should be something like this:
```
<a href="{{ 'home' | page }}">Home</a>
<a href="{{ 'contact' | page }}">Contact</a>
```

If these are Static Pages, use the `staticPage` filter instead.

**Comment by helinW** [comment]
I have used |page :
`<a href="{{'home'|page}}" id="pl-undefined">Home</a>`
`<a href="{{'contact'|page}}" id="pl-undefined">Contact</a>`

**Comment by helinW** [comment]
If i wrtie directly the url , I get a error 404 : 
https://ossmozbjj.com/fr/accueil

**Reply by mjauvin** [reply]
something is definitely not right.

are you using any other 3rd party plugins? Try disabling them.

**Comment by helinW** [comment, marked as answer]
Hello,
I made a new installation from scratch and just implement the "translate" plugin and it seems to work.
I don't know what the problem was.
Thanks for your help
